conda-recipes/llvmdev_manylinux/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set shortversion = "15.0" %}
+{% set version = "15.0.7" %}
+{% set sha256_llvm = "8b5fcb24b4128cf04df1b0b9410ce8b1a729cb3c544e6da885d234280dedeac6" %}
+{% set build_number = "0" %}
+
+package:
+  name: llvmdev
+  version: {{ version }}
+
+source:
+  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
+    sha256: {{ sha256_llvm }}
+    patches:
+    - ../llvm15-clear-gotoffsetmap.patch
+    - ../llvm15-remove-use-of-clonefile.patch
+    - ../llvm15-svml.patch
+    - ../compiler-rt-cfi-startproc-war.patch
+    - ../compiler-rt-macos-build.patch
```

Comment:
This looks like a `macos` patch? Since this is manylinux -- can that line be removed?

Reply:
This meta.yaml file is a copy of `../llvmdev/meta.yaml`. I will comment out lines that already have preprocessor conditions for non-linux. But, for patches, i will want to keep them the same as the non-manylinux version for ease of maintenance (so the code is the same in build variants)